v8/quotebacks/quotebacks_mdx.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Since you’re using third-party code, make sure to include a license header and a license file in the plugin directory. Mentioning the license in README would be a good idea as well.

Reply:
Done